ci/pipeline.yml

```diff
@@ -0,0 +1,119 @@
+---
+resource_types:
+  - name: gcs-resource
+    type: docker-image
+    source:
+      repository: frodenas/gcs-resource
+
+resources:
+  - name: gcp-service-broker
+    type: git
+    source:
+      uri: https://github.com/GoogleCloudPlatform/gcp-service-broker.git
+      branch: concourse
+      ignore_paths:
+        - ci/*
+
+  - name: gcp-service-broker-ci
```

Comment:
it's essentially a way to separate out the ci config as a resource. The fact that it's in the same repo is just how we've been setting things up, but really the code and the ci config should be considered different resources (which also allows for out-of-sync updating).